test/d2l-filter-dropdown/d2l-filter-dropdown-category.html

```diff
@@ -89,7 +89,7 @@
 						done();
 					});
 
-					var searchInputButton = categories[0].shadowRoot.querySelector('d2l-input-search').shadowRoot.querySelector('.d2l-input-search-search');
+					var searchInputButton = categories[0].shadowRoot.querySelector('d2l-input-search').shadowRoot.querySelector('d2l-button-icon:first-of-type');
 					searchInputButton.click();
```

Comment:
The search input has a `search()` method:
```suggestion
					categories[0].shadowRoot.querySelector('d2l-input-search').search();
```